buildozer/default.spec

```diff
@@ -99,6 +99,9 @@ fullscreen = 0
 # (str) Android NDK version to use
 #android.ndk = 9c
 
+# (int) Android NDK API to use (optional). This is the minimum API your API will support. 
```

Comment:
API is repeated (second one should be app), but other than that great! :)

Reply:
Whoops!  Too much multi-tasking. Ok, fixed.